mitmproxy/addons/save.py

```diff
@@ -44,6 +46,17 @@ def start_stream_to_path(self, path, flt):
         self.stream = io.FilteredFlowWriter(f, flt)
         self.active_flows = set()
 
+    def add(self, flow):
+        try:
+            self.stream.add(flow)
+        except OSError as e:
+            if e.errno == errno.ENOSPC:
+                ctx.log.error("Exiting due to insufficient space on disk")
+                if isinstance(self.stream, io.FilteredFlowWriter):
```

Comment:
I think we can also remove this check? Is there a case where this method is called and `self.stream` is not a FilteredFlowWriter?

Reply:
Thinking about it now, it is not needed. This check is from one of my previous commits as it was breaking on of the tests. Ill remove it and re-run the tests to make sure nothing breaks.